Formula/frps.rb

```diff
@@ -0,0 +1,72 @@
+class Frps < Formula
+  desc "This is the server of frp"
+  homepage "https://github.com/fatedier/frp"
+  url "https://github.com/fatedier/frp.git",
+      :tag      => "v0.24.1",
```

Comment:
This is no longer the latest version.

Reply:
Since this pr lasts for 2 month, the frp package has been updated to 2.26.x. I think it is good to merge this pr and make it updated then.

Reply:
yes, updated to v0.26.0